doc/panes/outline.rst

```diff
@@ -0,0 +1,48 @@
+#######
+Outline
+#######
+
+The **Outline** pane allows you to view and navigate the functions, classes, methods, cells and comments in open files.
```

Comment:
```suggestion
The **Outline** pane allows you to view and navigate the functions, classes, methods, cells and comments in open Python files.
```

I don't know if we should clarify this given that the pane appears empty for other file types (e.g. C, C++, etc) by default.

Note: The Outline should be populated if users add language servers for other programming languages, but that's far from trivial and we haven't tested it.